docs/topics/vs-code-extension-interface.adoc

```diff
@@ -10,9 +10,9 @@
 
 The interface of the {ProductName} ({ProductShortName}) extension is designed to make it easier for you to find information and perform actions:
 
-* In the left pane, you can see a directory tree named *Analysis Results* with a report icon at its top. You can click the icon to open the MTA report in your browser. Beneath the report icon are the other elements of the tree: the applications analyzed by {ProductShortName}, the rulesets used, and the issues discovered by the analysis.
+* In the left pane, you can see a directory tree named *Analysis Results* with a report icon at its top. You can click the icon to open the {ProductShortName} report in your browser. Beneath the report icon are the other elements of the tree: the applications analyzed by {ProductShortName}, the rulesets used, and the issues discovered by the analysis.
```

Comment:
Did QE check?

Reply:
@RichardHoch They only see the attribute.